pvlib/horizon.py

```diff
@@ -0,0 +1,646 @@
+"""
+The ``horizon`` module contains functions for horizon profile modeling.
+There are various geometric utilities that are useful in horizon calculations
+as well as a method that uses the googlemaps elevation API to create a
+horizon profile.
+"""
+from __future__ import division
+
+import itertools
+
+import numpy as np
+import warnings
+
+from scipy.interpolate import RegularGridInterpolator
```

Comment:
Since this is the only use of scipy, I suggest we move the import inside a try/except in `sample_using_interpolation`  See https://github.com/pvlib/pvlib-python/blob/0a14b273517a082603cc157faa88a5ab0ac4cac9/pvlib/solarposition.py#L887 for an example